lib/improver/tests/cube_combiner/test_CubeCombiner.py

```diff
@@ -211,15 +157,36 @@ def test_mean_multi_cube(self):
         self.assertEqual(result.name(), 'new_cube_name')
         self.assertArrayAlmostEqual(result.data, expected_data)
 
+    def test_with_mask(self):
+        """Test that the plugin preserves the mask if any of the inputs are
+        masked"""
+        expected_data = np.full((1, 2, 2), 1.2, dtype=np.float32)
+        mask = [[[False, True], [False, False]]]
+        self.cube1.data = np.ma.MaskedArray(self.cube1.data, mask=mask)
+        plugin = CubeCombiner('add')
+        result = plugin.process(
+            [self.cube1, self.cube2, self.cube3], 'new_cube_name')
+        self.assertIsInstance(result.data, np.ma.MaskedArray)
+        self.assertArrayAlmostEqual(result.data.data, expected_data)
+        self.assertArrayEqual(result.data.mask, mask)
+
```

Comment:
Do you think the `test_exception_for_cube_passed in` is relevant now that the raising of the exception within `CubeCombiner` has been removed?

Reply:
No, not really.  In fact I had to figure out why it still passes (it's because a "cube" isn't iterable and so has no `len`).  I'll get rid of this test.